source/importers/SynchronizeColorsTokens.ts

```diff
@@ -0,0 +1,243 @@
+	let resultColorsJson: Dict<Array<IResultFileJsonObject>> = {};
+
+	result.push(':root {');
+
+	jsonColors.forEach((obj) => {
+		const patternColorsToProcess = /RAW\/|theme/i;
+		const patternColorsToIgnore = /deprecated|Pattern/i;
+		const patternTheme = /theme/i;
+		if (
+			obj.name.match(patternColorsToProcess) &&
+			obj.name.match(patternColorsToIgnore) === null
+		) {
+			const nameSplit = obj.name.split('/');
+			let colorName = nameSplit[2] === undefined ? nameSplit[1] : nameSplit[2];
+			obj.values.hex = makeHexShortcut(obj.values.hex);
+
+			const colorFinalName = obj.name.match(patternTheme)
+				? '--' + colorName
+				: '--raw-' + colorName;
+
+			if (hexToCssVariable[obj.values.hex] === undefined) {
+				hexToCssVariable[obj.values.hex] = colorFinalName;
+			}
+
+			result.push(colorFinalName + ': ' + obj.values.hex + ';');
+
+			const rgb = hexToRgb(obj.values.hex);
+			if (rgb === null) {
+				throw new Error('ERROR! Cant convert this hex to rgb: ' + obj.values.hex);
+			}
+
+			result.push(colorFinalName + '-rgb' + ': ' + rgb + ';');
+
+			/** JSON object structure */
+			const colorNameSplitted = colorName.split('-');
+			const category = colorNameSplitted[1] === undefined ? 'default' : colorNameSplitted[0];
+			const resultJsonObject: IResultJsonObject = {
+				id: binValues.destination + '_' + colorName,
+				label: colorName,
+				value: obj.values.hex,
+				weight: parseInt(colorNameSplitted[1]),
+			};
+
+			if (temporaryColorsJson[category] === undefined) {
+				temporaryColorsJson[category] = [];
+			}
+			temporaryColorsJson[category].push(resultJsonObject);
+		}
+	});
+
+	result.push('}');
+
+	if (result.length === 2) {
+		throw new Error('ERROR! No colors to save');
+	}
+
+	Object.entries<Array<IResultJsonObject>>(temporaryColorsJson).forEach(([category, colors]) => {
+		Object.entries(colors).forEach(([, color]) => {
+			const resultFileJsonObject: IResultFileJsonObject = {
+				id: color.id,
+				label: color.label,
+				value: color.value,
+			};
+
+			if (resultColorsJson[category] === undefined) {
+				resultColorsJson[category] = [];
+			}
+			resultColorsJson[category].push(resultFileJsonObject);
+		});
+	});
+
+	arrayToFile(tokensFilesConfig.destinationPath + binValues.destination, result);
+	jsonToFile(tokensFilesConfig.destinationPath + binValues.destinationJson, resultColorsJson);
+
+	return hexToCssVariable;
+};
+
+const ImportSingleTokenFile = (
+	binValues: configFileObject,
+	jsonColors: Array<any>,
+	hexToCssVariable: Dict<string>,
+) => {
+	let result: Array<string> = [];
+	let resultJson = {};
+
+	jsonColors.forEach((obj) => {
+		const patternColorsToIgnore = /deprecated|Pattern|RAW|theme/i;
+		if (obj.name.match(patternColorsToIgnore) === null) {
+			let tokenName = obj.name;
+			tokenName = tokenName.replace(/\//i, '-');
+			obj.values.hex = makeHexShortcut(obj.values.hex);
+
+			if (obj.values.alpha !== 1) {
+				result.push(
+					'$' +
```

Comment:
I'm wondering if it's ok to define tokens as SASS variables. For it, it doesn't really matter now and we can migrate that in future, but for UX it can be important.

Let's consider alternative approach: define css variable for token and then define SASS variable to that Css variable
eg. 
```
--default-border: var(--raw-black);
$default-border: var(--default-border);
```

In this scenario, UX will be able to see token name used in CSS when using devtools inspect.

What do you think about it?

Reply:
It's done.